packages/mdc-tab-scroller/foundation.js

```diff
@@ -167,6 +172,14 @@ class MDCTabScrollerFoundation extends MDCFoundation {
     return this.rtlScrollerInstance_;
   }
 
+  /**
+   * Returns the width of the scroll content
+   * @return {number}
+   */
+  getScrollContentWidth() {
```

Comment:
I agree with Ken on this one. I don't think this method is needed, and can just be a one liner in the component.

Reply:
I don't disagree. Since this is a subcomponent that's required inside of another, we have yet to figure out a pattern for accessing the child elements of a subcomponent.